### ISSUE TITLE
Question VALARM

### ISSUE THREAD

**Issue by YM-KA** [adamgibbons/ics#124]
Hi, thanks for a great package! 
I am wondering about the <code>BEGIN:VALARM</code> tag. 
I have set an alarm, but Outlook (PC) and iCalendar (Mac) seems to ignore the alarm altogether. 

Is this a known problem with Outlook / iCalendar ? 
The format seems to be valid. 
Are any additional values needed for each client to understand/accept the alert? 

```
BEGIN:VCALENDAR
VERSION:2.0
CALSCALE:GREGORIAN
PRODID:adamgibbons/ics
METHOD:PUBLISH
X-PUBLISHED-TTL:PT1H
BEGIN:VEVENT
UID:-LovTg9y9HPDCHu8mVnD
SUMMARY:Purjolök
DTSTAMP:20191026T171500Z
DTSTART:20191024T080000Z
DESCRIPTION:En grönsak i lökfamiljen
URL:https://app.myapp.se/r/edit/-LovTg9y9HPDCHu8mVnD
STATUS:CONFIRMED
CATEGORIES:Matlista
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
BEGIN:VALARM
ACTION:DISPLAY
DESCRIPTION:En grönsak i lökfamiljen
TRIGGER:-PT10M
END:VALARM
DURATION:PT30M
END:VEVENT
END:VCALENDAR
```

**Comment by Deepakmukka1** [comment]
@adamgibbons its not even working in google calendar , the alarm is being ignored

**Comment by adamgibbons** [comment]
Please reopen if not fixed by #220.
